apps/env/env-1.py

```diff
@@ -114,7 +114,7 @@ def _run( self, args ) :
 		command = list( args["arguments"][i:] )
 		if command :
 			try :
-				return subprocess.call( command, env=env )
+				return subprocess.call( command, env=env, shell=False if os.name != "nt" else True )
```

Comment:
Could launching via a shell here mean that the shell's initialisation messes with the environment also, thus yielding a different environment than expected? If so, perhaps we should just adjust the tests to use another command?

Reply:
Yes, I guess using `shell` isn't quite as reliable. I changed the test that required the built-in shell command on Windows in 0afe708f883c00a1fa44f150c882e2bb0009f916. It uses Python now to directly query the environment variables of interest.